functions/cors-python/handler.py

```diff
@@ -12,3 +12,7 @@ def handle(event, context):
             "Content-Type": ["text/plain"],
         },
     }
+
+if __name__ == "__main__":
+    from scaleway_functions_python import local
+    local.serve_handler(handle)
```

Comment:
It's not important, but I like when files end with a blank line. For Python especially, it's an indication the file wasn't formatted with black.

Reply:
Done